must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -42,6 +42,12 @@ commands_get+=("events")
 commands_get+=("all")
 commands_get+=("role")
 commands_get+=("rolebinding")
+commands_get+=("volumesnapshot -A")
+commands_get+=("volumesnapshotclass")
+commands_get+=("volumesnapshotcontent")
+commands_get+=("volumesnapshot -A -o yaml")
+commands_get+=("volumesnapshotclass -o yaml")
+commands_get+=("volumesnapshotcontent -o yaml")
```

Comment:
add these in describe also and yaml also

Reply:
Done